scripts/change-log-generator.js

```diff
@@ -63,15 +63,19 @@ const PACKAGES = 'PACKAGES';
 const RELEASE_REGEX = new RegExp(/^origin\/release\/v\d{2}\.\d{1,2}\.\d/);
 const PR_REGEX = new RegExp(/(\(#\d+\))/);
 const COMMIT_REGEX = new RegExp(/^([\da-zA-Z]+)/);
+const TYPE_REGEX = new RegExp(/([a-zA-Z]+)(?:\([a-zA-Z]+\))?:/);
 
 /**
  * Checks if the user has provided a release branch override. If they
  * have not, returns the latest release branch.
  */
 function getReleaseBranch() {
+  if (ADD_VERBOSE_LOGGING) {
+    console.log('\nStep 1: Determine release branch.');
+  }
   var releaseIndex = process.argv.indexOf('-r');
   var releaseBranch =
-    releaseIndex > -1
+    releaseIndex > -1 && process.argv[releaseIndex + 1]
```

Comment:
This piece was needed since the task will now supply the argument for -r, when a user might not enter a value for it.